src/EnergyPlus/ExhaustAirSystemManager.hh

```diff
@@ -62,68 +62,61 @@ namespace ExhaustAirSystemManager {
     struct ExhaustAir
     {
         // Members
-        std::string Name;
+        std::string Name = "";
 
-        int AvailScheduleNum;
-        std::string ZoneMixerName;
-        int ZoneMixerIndex;
-        int CentralFanTypeNum;
-        std::string CentralFanName;
-        int CentralFanIndex;
+        int AvailScheduleNum = DataGlobalConstants::ScheduleAlwaysOn;
+        std::string ZoneMixerName = "";
+        int ZoneMixerIndex = 0;
+        int CentralFanTypeNum = 0;
+        std::string CentralFanName = "";
+        int CentralFanIndex = 0;
 
-        bool SizingFlag;
+        bool SizingFlag = true;
 
         // output variables
-        Real64 centralFan_MassFlowRate;
-        Real64 centralFan_VolumeFlowRate_Std;
-        Real64 centralFan_VolumeFlowRate_Cur;
-        Real64 centralFan_Power;
-        Real64 centralFan_Energy;
+        Real64 centralFan_MassFlowRate = 0.0;
+        Real64 centralFan_VolumeFlowRate_Std = 0.0;
+        Real64 centralFan_VolumeFlowRate_Cur = 0.0;
+        Real64 centralFan_Power = 0.0;
+        Real64 centralFan_Energy = 0.0;
 
         // Output acc variable for heat rejection outputs
-        Real64 exhTotalHVACReliefHeatLoss; // feed to state.dataHeatBal->SysTotalHVACReliefHeatLoss
-
-        // Default Constructor
-        ExhaustAir()
-            : AvailScheduleNum(0), ZoneMixerName(""), ZoneMixerIndex(0), CentralFanTypeNum(0), CentralFanName(""), CentralFanIndex(0),
-              SizingFlag(true), centralFan_MassFlowRate(0.0), centralFan_VolumeFlowRate_Std(0.0), centralFan_VolumeFlowRate_Cur(0.0),
-              centralFan_Power(0.0), centralFan_Energy(0.0), exhTotalHVACReliefHeatLoss(0.0)
-        {
-        }
+        Real64 exhTotalHVACReliefHeatLoss = 0.0; // feed to state.dataHeatBal->SysTotalHVACReliefHeatLoss
     };
 
     struct ZoneExhaustControl
     {
-        std::string Name;
-
-        int AvailScheduleNum;
-
-        std::string ZoneName;
-        int ZoneNum;
-        int ControlledZoneNum;
-
-        int InletNodeNum;
-        int OutletNodeNum;
-
-        Real64 DesignExhaustFlowRate;
-        int FlowControlTypeNum;
-        int ExhaustFlowFractionScheduleNum;
-        int SupplyNodeOrNodelistNum;
-        int MinZoneTempLimitScheduleNum;
-        int MinExhFlowFracScheduleNum;
-        int BalancedExhFracScheduleNum;
-        Real64 BalancedFlow;
-        Real64 UnbalancedFlow;
+        enum class FlowControlType
```

Comment:
What is our convention for declaring these enumerations inside the `struct`/`class` vs. just inside the namespace?

Reply:
Probably it does not have to be in one particular way over the other. But in general, I found it seems a reasonable balance to have the enum definition placed inside the struct that it is to be used, usually in the `.hh` file; and then have the string_view _UCs placed just inside the name space, usually in the `.cc` file to limit scope. The former will help to keep it in necessary scope and easy to find or being moved around with the entire struct in one piece. The latter will help to limit the reduce unnecessary visibility to this particular `.cc` file.

Reply:
:+1: